arch/arm64/src/imx9/imx9_usbdev.c

```diff
@@ -60,7 +60,7 @@
 #endif
 
 #define DCACHE_LINEMASK (ARMV8A_DCACHE_LINESIZE - 1)
-
+#define CACHE_ALIGN_UP(a)   (((a) + DCACHE_LINEMASK) & ~DCACHE_LINEMASK)
```

Comment:
Should we change this to DCACHE_ALIGN_UP to keep naming consistent?

Reply:
sure